next/services/meili/meiliTypes.ts

```diff
@@ -47,7 +47,17 @@ export type DocumentMeili = Omit<Document, '__typename' | 'documentCategory' | '
   file: Omit<UploadFile, '__typename'>
 }
 
-export type DisclosureMeili = Omit<Disclosure, '__typename' | 'files'> & {
+/**
+ * Enum generated by Strapi doesn't contain accents.
+ */
+export enum DisclosureTypeFixed {
```

Comment:
Hmmmm yes, they don't have a human-readable value, they should be used just as keys.